src/cargo/ops/registry.rs

```diff
@@ -367,8 +367,12 @@ pub fn configure_http_handle(config: &Config, handle: &mut Easy) -> CargoResult<
     // connect phase as well as a "low speed" timeout so if we don't receive
     // many bytes in a large-ish period of time then we time out.
     handle.connect_timeout(Duration::new(30, 0))?;
-    handle.low_speed_limit(10 /* bytes per second */)?;
-    handle.low_speed_time(Duration::new(30, 0))?;
+    if let Some(config_low_speed_limit) = low_speed_limit(config)? {
+        handle.low_speed_limit(config_low_speed_limit)?;
+    }
+    if let Some(config_low_speed_time) = low_speed_time(config)? {
+        handle.low_speed_time(config_low_speed_time)?;
```

Comment:
I think this conflicts with `http_timeout` below, but could it suffice to avoid configuring this manually and letting `http_timeout` below configure it?

Reply:
Right. I just didn't notice the option was configured with the http-timeout.